common/config.go

```diff
@@ -59,7 +59,7 @@ type Config struct {
 	VaultKeyID        *uuid.UUID `json:"vault_key_id"`
 	VaultRefreshToken *string    `json:"-"`
 
-	ProvideRefreshToken    *string `json:"-"`
+	ProvideAccessToken    *string `json:"-"`
```

Comment:
By providing the refresh token, you actually can authorize both new access and refresh tokens. Why change this to access token? It limits the node to only 24 hours of operability... @bitwiseguy